src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -70,6 +70,8 @@ def update_data(self):
             if bridge_port_id not in self.if_bpid_map:
                 continue
             port_id = self.if_bpid_map[bridge_port_id]
+            if port_id not in self.if_id_map:
```

Comment:
>port_id not in self.if_id_map: [](http://example.com/codeflow?start=15&length=30)

What is the root cause of this issue?

Reply:
Could you give more description about your topology and config? I guess your DUT has a PortChannel in Vlan, and there are traffic hit the port in PortChannel. If this is true, we recently fixed it by [https://github.com/Azure/sonic-snmpagent/pull/203](https://github.com/Azure/sonic-snmpagent/pull/203). Could you try latest master code?